### PR TITLE
Fixes #132: Added Documentation regarding Contributing

### LINKED ISSUE #13: No Documentation regarding Contributing
It would be great if you could find Documentation regarding Contributing to the Repository. So that a Contributor who is willing to contribute, can get a clear idea of Contributing to the Project.

### LINKED ISSUE #132: Controls disappear automatically
### Expected results

I would like to switch on the controls and remove them manually.

### Actual results

After some time, e.g. when a dialog pop up, the controls disappear.

### Steps to reproduce the issue

1. In the bottom right corner click the running person.
2. the controls open
3. wait some time or navigate
4. the controls disappear

If you also think they should stay so we can play the game with touch, you can solve this :)

### PR DESCRIPTION
**Pull request by naveentherocker**
Added README.md and CONTRIBUTING.md regarding Contributing to the Project.
Fixes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,30 @@ Implementation
...
+
+Contributions, Bug Reports, Feature Requests
+--------------
+This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests by submitting pull requests as well. Please report issues in the [GitHub tracker](https://github.com/fossasia/labyrinth/issues/new).
+
+## Issue and Branch Policy
+
+Before making a pull request, please file an issue. So, other developers have the chance to give feedback or discuss details. Match every pull request with an issue please and add the issue number in description e.g. like "Fixes #123".
+
+We have the following branches
```
Comment (xeon-zolt) [inline README.md]: Are you sure?

File: README.md [hunk 2]
```diff
@@ -31,3 +31,30 @@ Implementation
...
+Contributions, Bug Reports, Feature Requests
+--------------
+This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests by submitting pull requests as well. Please report issues in the [GitHub tracker](https://github.com/fossasia/labyrinth/issues/new).
+
+## Issue and Branch Policy
+
+Before making a pull request, please file an issue. So, other developers have the chance to give feedback or discuss details. Match every pull request with an issue please and add the issue number in description e.g. like "Fixes #123".
+
+We have the following branches   
+ * **development**
```
Comment (xeon-zolt) [inline README.md]: we don't have a development branch

**Comment by naveentherocker** [comment]
@xeon-zolt  Now it's fine right

**Comment by xeon-zolt** [comment]
Looks Good To Me 👍

**Pull request merged**